helpers/tools.ts

```diff
@@ -37,6 +38,7 @@ export const supportedTools: Tool[] = [
         version: "0.1.2",
       },
     ],
+    tsSupport: true,
```

Comment:
we might later also have tools that only work in TS, how about
```suggestion
    supportedLanguages: ["python","typescript"],
```